Kernel/Syscalls/mmap.cpp

```diff
@@ -563,14 +563,8 @@ ErrorOr<FlatPtr> Process::sys$allocate_tls(Userspace<char const*> initial_data,
 
         TRY(main_thread->make_thread_specific_region({}));
 
-#if ARCH(I386)
-        auto& tls_descriptor = Processor::current().get_gdt_entry(GDT_SELECTOR_TLS);
-        tls_descriptor.set_base(main_thread->thread_specific_data());
-        tls_descriptor.set_limit(main_thread->thread_specific_region_size());
-#else
         MSR fs_base_msr(MSR_FS_BASE);
```

Comment:
is it advisable to mark this here as `#if ARCH(X86_64)` with a ‘to doʼ for `ARCH(AARCH64)`? 🫢